modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/BasePython_conf.pm

```diff
@@ -43,6 +43,17 @@ sub default_options {
     }
 }
 
+sub beekeeper_extra_cmdline_options {
```

Comment:
Why do we need an extra BaseConf here? 

I wonder because we have quite the same code for the "Perl" one... which happens to be ready for slurm. This one doesn't seem to be.

Reply:
Solved by adding the python code to the pipeline itself and swapping to the base_conf